### DISCUSSION TITLE
v-popover component not working on v4

### DISCUSSION THREAD

**Discussion by dandyweng** [Akryum/v-tooltip#788] [Q&A]
Hi, I’m migrating my project to Vue 3 and I followed your [migration guide](https://github.com/Akryum/v-tooltip/discussions/603) to install `v-tooltip@vue3 v4.0.0-alpha.1`. I installed the plugin into Vue with:

```js
import vTooltip from "v-tooltip"

const app = createApp({})
app.use(vTooltip)
```

And I got warnings of `Failed to resolve component: v-popover`. I took a look at the install method and it seems the popover components were not installed:
https://github.com/Akryum/v-tooltip/blob/1b987c7a9db84407c6c9055be67bc093c42398b8/packages/v-tooltip/src/index.js#L38-L57

**Comment by Akryum** [comment]
![image](https://user-images.githubusercontent.com/2798204/130316817-06ca9c9a-c5c4-44af-a4b8-4cd126db86fa.png)

**Reply by dandyweng** [reply]
Well, I find the inconsistency of the component's name between the migration guide and the doc quite confusing. I thought a dropdown component is something else since it's called a popover component in the doc, even though they're technically the same thing :)

Thanks for your reply.

**Reply by Akryum** [reply]
[The docs](https://v-tooltip.netlify.app/guide/component.html) uses `<VDropdown/>`

**Comment by Akryum** [comment]
Migration guide is being worked on [here](https://v-tooltip.netlify.app/migration/migration-from-v2.html)
